src/pages/ReferenceGuidesLanding.spec.tsx

```diff
@@ -29,6 +29,15 @@ describe('ReferenceGuidesLanding', () => {
 					>
 						{messages.i18nDescription}
 					</LandingLink>
+					<LandingLink
+						{...{ '~key': 'stylingAndThemingLink' }}
```

Comment:
Is there any particular reason you write it this way instead of `assertion-key='stylingAndThemingLink'`?

Reply:
Not really 😄 . Don't think I had used `assertion-key` yet when I wrote this. Will update